contracts/sui/contracts/sources/stork.move

```diff
@@ -0,0 +1,172 @@
+module stork::stork {
+
+    // === Imports ===
+
+    use stork::admin::AdminCap;
+    use stork::state::{Self, StorkState};
+    use stork::temporal_numeric_value_feed::{Self, TemporalNumericValueFeed};
+    use stork::temporal_numeric_value::TemporalNumericValue;
+    use stork::encoded_asset_id::{Self, EncodedAssetId};
+    use stork::verify::verify_stork_evm_signature;
+    use stork::event::Self;
+    use stork::update_temporal_numeric_value_evm_input::UpdateTemporalNumericValueEvmInput;
+    use stork::update_temporal_numeric_value_evm_input_vec::UpdateTemporalNumericValueEvmInputVec;
+    use sui::sui::SUI;
+    use sui::coin::Coin;
+    use sui::object_table::ObjectTable;
+
+    // === Errors ===
+
+    const EInvalidSignature: u64 = 0;
+    const EInsufficientFee: u64 = 1;
+    const EFeedNotFound: u64 = 2;
+
+    // === Entry Functions ===
+
+    entry fun init_stork(
+        // admin capability
+        _: &AdminCap,
+        // the address of the Stork program
+        stork_sui_address: address,
+        // Storks EVM public key
+        stork_evm_public_key: vector<u8>,
+        // the fee to update a value
+        single_update_fee: u64,
+        // version of the Stork state 
+        stork_state_version: u64,
+        // context
+        ctx: &mut TxContext,
+    ) {
+        let state = state::new(stork_sui_address, stork_evm_public_key, single_update_fee, stork_state_version, ctx);
+        let state_id = object::id(&state);
+        state::share(state);
+        event::emit_stork_initialization_event(
+            stork_sui_address,
+            stork_evm_public_key,
+            single_update_fee,
+            state_id,
+            stork_state_version
+        );
+    }
+
+    // === Public Functions ===
+
+    // updates the price feed for an asset 
+    // input data is an assets update data signed with Storks EVM public key
+    public fun update_single_temporal_numeric_value_evm(
```

Comment:
Why did you want to keep this as well as the multiple update method?

Reply:
Mostly to leave it available as an option for us in the future or people who want to write their own pushers. Sort of in the mindset that its written, it works, if this was say a rest api endpoint it would feel natural to include it even if we don't personally use it in our clients, and because upgrading is a pain I'd rather not have to add it back in later. If you want to take it out we can, as we're not using it, however it feels natural to keep it in imo.